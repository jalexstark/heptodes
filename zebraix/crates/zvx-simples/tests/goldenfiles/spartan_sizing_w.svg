<?xml version="1.0" encoding="UTF-8"?>
<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="400pt" height="350pt" viewBox="0 0 400 350">
<path fill="none" stroke-width="1" stroke-linecap="butt" stroke-linejoin="miter" stroke="rgb(0%, 0%, 0%)" stroke-opacity="1" stroke-miterlimit="10" d="M 40.910156 334.089844 L 40.910156 15.910156 L 359.089844 15.910156 L 359.089844 334.089844 Z M 40.910156 334.089844 "/>
<path fill="none" stroke-width="0.45" stroke-linecap="butt" stroke-linejoin="miter" stroke="rgb(0%, 0%, 0%)" stroke-opacity="1" stroke-dasharray="4.5 3.15" stroke-miterlimit="10" d="M 40.910156 175 L 359.089844 175 M 40.910156 220.453125 L 359.089844 220.453125 M 40.910156 265.910156 L 359.089844 265.910156 M 40.910156 311.363281 L 359.089844 311.363281 M 40.910156 129.546875 L 359.089844 129.546875 M 40.910156 84.089844 L 359.089844 84.089844 M 40.910156 38.636719 L 359.089844 38.636719 "/>
<path fill="none" stroke-width="0.45" stroke-linecap="butt" stroke-linejoin="miter" stroke="rgb(0%, 0%, 0%)" stroke-opacity="1" stroke-dasharray="4.5 3.15" stroke-miterlimit="10" d="M 245.453125 334.089844 L 245.453125 15.910156 M 200 334.089844 L 200 15.910156 M 154.546875 334.089844 L 154.546875 15.910156 M 109.089844 334.089844 L 109.089844 15.910156 M 63.636719 334.089844 L 63.636719 15.910156 M 290.910156 334.089844 L 290.910156 15.910156 M 336.363281 334.089844 L 336.363281 15.910156 "/>
<path fill-rule="nonzero" fill="rgb(0%, 0%, 65%)" fill-opacity="1" d="M 248.03125 84.089844 C 248.03125 87.527344 242.878906 87.527344 242.878906 84.089844 C 242.878906 80.65625 248.03125 80.65625 248.03125 84.089844 "/>
<path fill-rule="nonzero" fill="rgb(0%, 0%, 65%)" fill-opacity="1" d="M 256.710938 105.03125 C 256.710938 108.464844 251.558594 108.464844 251.558594 105.03125 C 251.558594 101.59375 256.710938 101.59375 256.710938 105.03125 "/>
<path fill-rule="nonzero" fill="rgb(0%, 0%, 65%)" fill-opacity="1" d="M 260.65625 122.8125 C 260.65625 126.246094 255.503906 126.246094 255.503906 122.8125 C 255.503906 119.375 260.65625 119.375 260.65625 122.8125 "/>
<path fill-rule="nonzero" fill="rgb(0%, 0%, 65%)" fill-opacity="1" d="M 260.816406 138.066406 C 260.816406 141.503906 255.664062 141.503906 255.664062 138.066406 C 255.664062 134.632812 260.816406 134.632812 260.816406 138.066406 "/>
<path fill-rule="nonzero" fill="rgb(0%, 0%, 65%)" fill-opacity="1" d="M 258.132812 151.429688 C 258.132812 154.867188 252.980469 154.867188 252.980469 151.429688 C 252.980469 147.996094 258.132812 147.996094 258.132812 151.429688 "/>
<path fill-rule="nonzero" fill="rgb(0%, 0%, 65%)" fill-opacity="1" d="M 253.554688 163.53125 C 253.554688 166.964844 248.402344 166.964844 248.402344 163.53125 C 248.402344 160.097656 253.554688 160.097656 253.554688 163.53125 "/>
<path fill-rule="nonzero" fill="rgb(0%, 0%, 65%)" fill-opacity="1" d="M 248.03125 175 C 248.03125 178.433594 242.878906 178.433594 242.878906 175 C 242.878906 171.566406 248.03125 171.566406 248.03125 175 "/>
<path fill-rule="nonzero" fill="rgb(0%, 0%, 65%)" fill-opacity="1" d="M 242.507812 186.46875 C 242.507812 189.902344 237.355469 189.902344 237.355469 186.46875 C 237.355469 183.035156 242.507812 183.035156 242.507812 186.46875 "/>
<path fill-rule="nonzero" fill="rgb(0%, 0%, 65%)" fill-opacity="1" d="M 237.929688 198.570312 C 237.929688 202.003906 232.777344 202.003906 232.777344 198.570312 C 232.777344 195.132812 237.929688 195.132812 237.929688 198.570312 "/>
<path fill-rule="nonzero" fill="rgb(0%, 0%, 65%)" fill-opacity="1" d="M 235.246094 211.933594 C 235.246094 215.367188 230.09375 215.367188 230.09375 211.933594 C 230.09375 208.496094 235.246094 208.496094 235.246094 211.933594 "/>
<path fill-rule="nonzero" fill="rgb(0%, 0%, 65%)" fill-opacity="1" d="M 235.402344 227.1875 C 235.402344 230.625 230.253906 230.625 230.253906 227.1875 C 230.253906 223.753906 235.402344 223.753906 235.402344 227.1875 "/>
<path fill-rule="nonzero" fill="rgb(0%, 0%, 65%)" fill-opacity="1" d="M 239.351562 244.96875 C 239.351562 248.40625 234.199219 248.40625 234.199219 244.96875 C 234.199219 241.535156 239.351562 241.535156 239.351562 244.96875 "/>
<path fill-rule="nonzero" fill="rgb(0%, 0%, 65%)" fill-opacity="1" d="M 248.03125 265.910156 C 248.03125 269.34375 242.878906 269.34375 242.878906 265.910156 C 242.878906 262.472656 248.03125 262.472656 248.03125 265.910156 "/>
<path fill="none" stroke-width="1" stroke-linecap="butt" stroke-linejoin="miter" stroke="rgb(0%, 40%, 0%)" stroke-opacity="1" stroke-miterlimit="10" d="M 245.453125 84.089844 C 290.910156 175 200 175 245.453125 265.910156 "/>
</svg>
