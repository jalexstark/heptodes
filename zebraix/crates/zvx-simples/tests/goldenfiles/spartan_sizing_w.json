{
  "sizing_scheme": "SquareCenter",
  "base_height": 350.0,
  "axes_range": [
    -4.5,
    -3.5,
    2.5,
    3.5
  ],
  "padding": [
    0.05
  ],
  "drawables": [
    {
      "drawable": {
        "Polyline": {
          "segment_choices": {
            "closure": "Closes"
          },
          "locations": [
            [
              -4.5,
              -3.5
            ],
            [
              -4.5,
              3.5
            ],
            [
              2.5,
              3.5
            ],
            [
              2.5,
              -3.5
            ]
          ]
        }
      }
    },
    {
      "drawable": {
        "Lines": {
          "line_choice": "Light",
          "coords": [
            [
              [
                -4.5,
                0.0
              ],
              [
                2.5,
                0.0
              ]
            ]
          ],
          "offsets": [
            [
              0.0,
              0.0
            ],
            [
              -0.0,
              -1.0
            ],
            [
              -0.0,
              -2.0
            ],
            [
              -0.0,
              -3.0
            ],
            [
              0.0,
              1.0
            ],
            [
              0.0,
              2.0
            ],
            [
              0.0,
              3.0
            ]
          ]
        }
      }
    },
    {
      "drawable": {
        "Lines": {
          "line_choice": "Light",
          "coords": [
            [
              [
                0.0,
                -3.5
              ],
              [
                0.0,
                3.5
              ]
            ]
          ],
          "offsets": [
            [
              0.0,
              0.0
            ],
            [
              -1.0,
              -0.0
            ],
            [
              -2.0,
              -0.0
            ],
            [
              -3.0,
              -0.0
            ],
            [
              -4.0,
              -0.0
            ],
            [
              1.0,
              0.0
            ],
            [
              2.0,
              0.0
            ]
          ]
        }
      }
    },
    {
      "layer": 20,
      "drawable": {
        "Points": {
          "point_choice": "Dot",
          "color_choice": "Blue",
          "centers": [
            [
              0.0,
              2.0
            ],
            [
              0.1909722222222222,
              1.5393518518518519
            ],
            [
              0.2777777777777778,
              1.1481481481481481
            ],
            [
              0.28125,
              0.8125
            ],
            [
              0.2222222222222222,
              0.5185185185185185
            ],
            [
              0.12152777777777778,
              0.2523148148148148
            ],
            [
              0.0,
              0.0
            ],
            [
              -0.12152777777777778,
              -0.2523148148148148
            ],
            [
              -0.2222222222222222,
              -0.5185185185185185
            ],
            [
              -0.28125,
              -0.8125
            ],
            [
              -0.2777777777777778,
              -1.1481481481481481
            ],
            [
              -0.1909722222222222,
              -1.5393518518518519
            ],
            [
              0.0,
              -2.0
            ]
          ]
        }
      }
    },
    {
      "layer": 30,
      "drawable": {
        "Cubic": {
          "segment_choices": {
            "color": "Green"
          },
          "c": [
            [
              0.0,
              2.0
            ],
            [
              1.0,
              0.0
            ],
            [
              -1.0,
              0.0
            ],
            [
              0.0,
              -2.0
            ]
          ]
        }
      }
    }
  ]
}
